core/src/test/java/com/netflix/conductor/core/execution/tasks/DoWhileTest.java

```diff
@@ -107,6 +107,7 @@ public void setup() {
         loopTask.setIteration(1);
         loopWorkflowTask = new WorkflowTask();
         loopWorkflowTask.setTaskReferenceName("loopTask");
+        loopWorkflowTask.setType(TaskType.DO_WHILE.name());
```

Comment:
why was the task type removed?

Reply:
It was never added. The tests were running fine earlier because I was directly using DoWhile class. Now we need because I have changed the way loopOvertask is being calculated. Earlier the approach was find all the task which has taskReferenceName as ```___iteration``` but now approach is the DO_WHILE task must ```has()``` that task.